mailbox/plugin/quota-search-elasticsearch-v6/src/main/java/org/apache/james/quota/search/elasticsearch/ElasticSearchQuotaSearcher.java

```diff
@@ -57,34 +54,31 @@ public ElasticSearchQuotaSearcher(RestHighLevelClient client, ReadAliasName read
 
     @Override
     public List<User> search(QuotaQuery query) {
-        Stream<User> results = new ScrollIterable(client, prepareSearch(query))
-            .stream()
-            .flatMap(searchResponse -> Arrays.stream(searchResponse.getHits()
-                .getHits()))
-            .map(SearchHit::getId)
-            .map(User::fromUsername)
-            .skip(query.getOffset().getValue());
-
-        return query.getLimit().getValue()
-            .map(results::limit)
-            .orElse(results)
-            .collect(Guavate.toImmutableList());
+        try {
+            return Arrays.stream(client.search(prepareSearch(query), RequestOptions.DEFAULT)
+                .getHits()
+                .getHits())
```

Comment:
It looks weird, can you find a better name?

Reply:
For what?

Reply:
FYI: `.getHits()` returns a `Hits` instance. `Hits.getHits()` returns an array `Hit []`

Reply:
That's ES API... not ours :trollface: